### PR TITLE
Bubble sort

### PR DESCRIPTION
**Pull request by ss18**
- Bubble sort added;
-  Readme sort sort updated: sorted alphabetically, heap_sort was missing;

### CODE REVIEW

File: sort/bubble_sort.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    n = len(arr)
+    swapped = True
+    while swapped:
+        swapped = False
+        for i in range(1, n - 1):
+            if arr[i - 1] > arr[i]:
+                swap(i - 1, i)
+                swapped = True
+
+array = [1, 5, 65, 23, 57, 1232, -1, -5, -2, 242, 100, 4, 423, 2, 564, 9, 0, 10, 43, 64]
```
Comment (ankit167) [inline sort/bubble_sort.py]: The number of characters exceed 80

File: sort/bubble_sort.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+"""
+https://en.wikipedia.org/wiki/Bubble_sort
+
+Complexity: O(N^2)
+
+"""
+
```
Comment (ankit167) [inline sort/bubble_sort.py]: Add extra blank line.

**Comment by ss18** [comment]
@ankit167 fixed review comments

**Comment by ankit167** [comment]
@ss18 Input [3,2,1] is giving wrong output- [2,3,1]

**Comment by ss18** [comment]
@ankit167 fixed

**Pull request merged**